### PR TITLE
moe: submission

### PR DESCRIPTION
**Pull request by aeiouaeiouaeiouaeiouaeiouaeiou**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.11.6 15G22010 x86_64
Xcode 8.2.1 8C1002

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: editors/moe/Portfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                moe
+version             1.12
+categories          editors
```
Comment (herbygillot) [inline editors/moe/Portfile:7]: Can we add a `revision` field?
```
version             1.12
revision            0
```

File: editors/moe/Portfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+platforms           darwin
+license             GPL-3
+maintainers         {aeiouaeiouaeiouaeiouaeiouaeiou @aeiouaeiouaeiouaeiouaeiouaeiou} openmaintainer
```
Comment (pmetzger) [inline editors/moe/Portfile:10]: You don't have a macports.org email address so you can't use shorthand here and need to list your email address as described in the MacPorts documentation.

**Review by pmetzger — CHANGES_REQUESTED** [review]
Looks mostly fine, only one substantive comment.

File: editors/moe/Portfile (hunk 1 above)
Comment (aeiouaeiouaeiouaeiouaeiouaeiou) [reply editors/moe/Portfile:9]: Done.

**Pull request merged**